server/etcdserver/apply/metrics.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apply
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	alarms = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "etcd_debugging",
```

Comment:
Is there any reason why these metrics are in debugging namespace?

Reply:
I don't known, if we add new metrics, shoud it address in debugging firstly?

Reply:
I'm not sure on when it's named `debugging` and when it's not
but this metric would definitely be useful when running etcd in production, and I'm not sure if it's useful for debugging.

Reply:
@serathius please point out should this metrics start with `etcd_debugging`?

Reply:
Yes, that's fine and it should. Any new metrics are done that way. Eventually, if it's stable, it can be moved to `etcd`.